samples/package.json

```diff
@@ -14,6 +14,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "google-auth-library": "^7.1.0",
+    "googleapis": "^73.0.0",
```

Comment:
Could I get you to try out https://www.npmjs.com/package/@googleapis/iam, I love to stress test our individual modules, and it should improve install size.

Reply:
Thanks for your feedback. In externalclient-setup.js, I still need to set auth with import from **googleapis**,
`google.options({auth});`
I am not able to setup the auth if only install `@googleapis/iam` for this case.

Reply:
won't this work?

```js
const Iam = require('@googleapis/iam')
const iam = await iam.iam({
    version: 'v1',
    auth
});
const response = await iam.projects.locations.workloadIdentityPools.create({});
```

You can pass auth to the client itself, it doesn't need to be configured globally (_perhaps I'm missing the issue though._).

Reply:
yes, this is working. Thank you.